pkg/observ/stats.go

```diff
@@ -15,11 +18,21 @@ import (
 // Currently it supports: prometheus
 func RegisterStatsExporter(r *mux.Router, statsExporter, service string) (func(), error) {
 	const op errors.Op = "observ.RegisterStatsExporter"
+	var stop = func() {}
+	var err error
```

Comment:
@fedepaol can you define the error inside of the `case` statements below? I think it would be a bit easier to follow that way

Reply:
Nevermind, that ends up looking messier. Ignore me!